Sources/CombineExternalAccessory/UnconfiguredExternalAccessory.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  UnconfiguredExternalAccessory.swift
+//  WACTest
+//
+//  Created by Luis Reisewitz on 06.08.20.
+//  Copyright © 2020 Lautsprecher Teufel GmbH. All rights reserved.
+//
+
+import Combine
+import ExternalAccessory
+import Foundation
+import UIKit
+
+public struct UnconfiguredExternalAccessory {
+    /// Browser to use for configuring. We create a new one here, as we need to set the delegate.
+    /// Switching the delegate of the existing Browser used for discovering devices would hang the discovery
+    /// Publisher.
+    let browser = EAWiFiUnconfiguredAccessoryBrowser()
+    let accessory: EAWiFiUnconfiguredAccessory
+
+    // TODO: [COMBINE]: convert to promise later with foundationextension
+    /// - Warning: This completes the browser's search process and terminates the `Publisher`.
+    public func configure(on viewController: UIViewController) -> AnyPublisher<Void, ConfigurationError> {
```

Comment:
If you don't mind making UnconfiguredExternalAccessoryPublisher public, you could return it here instead of erasing. This is totally optional, but I think it makes the API more clear. UnconfiguredExternalAccessoryPublisher can have a internal init, so nobody can create it from outside, but see it shouldn't be a problem.